app/assets/javascripts/app/models/post/interactions.js

```diff
@@ -44,12 +44,16 @@ app.models.Post.Interactions = Backbone.Model.extend({
   },
 
   userLike : function(){
-    return this.likes.select(function(like){ return like.get("author").guid === app.currentUser.get("guid")})[0];
+    return this.likes.select(function(like){
+      if(like.get("author") !== undefined) {
```

Comment:
In which case the author can be undefined? That looks like a bug in the creation of the like object.